tasks/build/browserify.js

```diff
@@ -30,9 +32,14 @@ module.exports = function(grunt) {
       // Render the banner for the top of the file
       const banner = grunt.template.process(bannerTemplate);
 
+      let globalsVars = {};
```

Comment:
`globalsVars` or `globals` or `globalVars`?

Reply:
Ah. Should be `globalVars`, based on the browserify option `insertGlobalVars`. I'll correct that.